src/fragments/lib/auth/js/emailpassword.mdx

```diff
@@ -2,6 +2,43 @@
 
 Create a new user in the Amazon Cognito UserPool by passing the new user's email address, password, and other attributes to `Auth.signUp`.
 
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+interface SignUpParams {
+  username: string;
+  password: string;
+  email: string;
+  phoneNumber: string;
+}
+
+export async function signUp(signUpParams: SignUpParams) {
```

Comment:
Can you update this to include passing in `username`, `password`, `email`,  and `phoneNumber` as a destructured object such as below:

`signUp({username, password, email, phoneNumber}:SignUpParameters)`